app/models/manageiq/providers/openstack/base_metrics_capture.rb

```diff
@@ -1,4 +1,4 @@
-class ManageIQ::Providers::Openstack::BaseMetricsCapture < ManageIQ::Providers::BaseManager::MetricsCapture
+class ManageIQ::Providers::Openstack::BaseMetricsCapture < ManageIQ::Providers::CloudManager::MetricsCapture
```

Comment:
Hm actually just noticed this class is used by the Cloud, Infra, and NetworkManagers.  I think this should become a mixin and those classes should then inherit from the relevant base classes.